turbinia/evidence.py

```diff
@@ -374,3 +433,46 @@ def __init__(self, artifact_name=None):
     super(ExportedFileArtifact, self).__init__()
     self.artifact_name = artifact_name
     self.copyable = True
+
+
+class DockerContainer(Evidence):
+  """Evidence object for a DockerContainer filesystem.
+
+  Attributes:
+    container_id(str): The ID of the container to mount.
+    _container_fs_path(str): Full path to where the container filesystem will
+      be mounted.
+    _docker_root_directory(str): Full path to the docker root directory.
+  """
+
+  # ABSOLUTELY NO LEADING / HERE
+  DEFAULT_DOCKER_DIRECTORY_PATH = 'var/lib/docker'
+
+  def __init__(self, container_id=None, *args, **kwargs):
+    """Initialization for Docker Container."""
+    super(DockerContainer, self).__init__(*args, **kwargs)
+    self.container_id = container_id
+    self._container_fs_path = None
+    self._docker_root_directory = None
+
+    self.context_dependent = True
+
+  def _preprocess(self):
+    if not self.parent_evidence:
+      raise TurbiniaException(
+          'Evidence of type DockerContainer should have a parent_evidence set.')
+    if not self.parent_evidence.is_mounted:
+      raise TurbiniaException(
+          'Parent Evidence ({0}) does not have a mounted filesystem'.format(
+              self.parent_evidence))
```

Comment:
maybe self.parent_evidence.name?  Otherwise you can do {0!s} above and I think it will work.